src/resources/js/views/month-mobile-events.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * Makes sure we have all the required levels on the Tribe Object
+ *
+ * @since TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events = tribe.events || {};
+tribe.events.views = tribe.events.views || {};
+
+/**
+ * Configures Month Mobile Events Object in the Global Tribe variable
+ *
+ * @since TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events.views.monthMobileEvents = {};
+
+/**
+ * Initializes in a Strict env the code that manages the Event Views
+ *
+ * @since TBD
+ *
+ * @param  {PlainObject} $   jQuery
+ * @param  {PlainObject} obj tribe.events.views.manager
+ *
+ * @return {void}
+ */
+( function( $, obj ) {
+	'use strict';
+	var $document = $( document );
+
+	/**
+	 * Selectors used for configuration and setup
+	 *
+	 * @since TBD
+	 *
+	 * @type {PlainObject}
+	 */
+	obj.selectors = {
+		calendar: '.tribe-events-calendar-month',
+		calendarDay: '.tribe-events-calendar-month__day-cell--mobile',
+		calendarDaySelected: '.tribe-events-calendar-month__day-cell--selected',
+		mobileEventsDay: '.tribe-events-calendar-month-mobile-events__mobile-day',
+	};
+
+	/**
+	 * Closes all mobile events
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $container jQuery object of view container
+	 *
+	 * @return {void}
+	 */
+	obj.closeAllEvents = function( $container ) {
+		$container.find( obj.selectors.calendarDay ).each( function( index, header ) {
+			var $header = $( header );
+			var contentId = $header.attr( 'aria-controls' );
```

Comment:
Same as the above, would love to see this being selector not a ID only kinda thing.